be/src/runtime/data_stream_sender.cpp

```diff
@@ -386,7 +386,8 @@ DataStreamSender::DataStreamSender(RuntimeState* state, int sender_id, const Row
           _bytes_sent_counter(nullptr),
           _dest_node_id(sink.dest_node_id),
           _destinations(destinations),
-          _enable_exchange_pass_through(enable_exchange_pass_through) {
+          _enable_exchange_pass_through(enable_exchange_pass_through),
+          _output_columns(sink.output_columns) {
```

Comment:
Maybe this is non-pipeline version? Maybe we don't need it.

Reply:
Yeah, you are right, don't need modify it